LondonBritishLibrary/add/BLadd24992.xml

```diff
@@ -39,57 +39,64 @@ schematypens="http://relaxng.org/ns/structure/1.0"
 
                     <msContents>
                         <summary/>
-
+                        
                         <msItem xml:id="ms_i1">
+                            <locus from="2r"/>
+                            <title type="complete" ref="LIT2701Dawit"/>
+
+                        <msItem xml:id="ms_i1.1">
                             <locus from="2r"/>
```

Comment:
I am not sure that you should use LIT2701Dawit here, it is Mazmura Degel plus Weddase and Anqasa. Maybe we could create a LIT for MD accompanied by these texts, which is not so uncommon.

Reply:
I think you worked most on these texts, so please decide that. My thinking was that someone looking for Dāwits would want to find also those which include Mazmura dəngəl, but of course the same could be achieved with a new ID and the appropriate recensions.

Reply:
I see, let's leave it like that by now; I would need time to come with a further solution on MD.

Reply:
I meant relations instead of recensions in my last comment...